### DISCUSSION TITLE
news/2024/01/first-post/

### DISCUSSION THREAD

**Discussion by giscus** [userliluzibert/userliluzibert.github.io#2] [Giscus]
# news/2024/01/first-post/

Introduction
This is my first post of my new blog website, Blog Dude (shoutout to whoever knows which game I’m referring to). I couldn’t have done it without the help of Razon Yang, the wonderful developer of the Hugo Bootstrap theme. He has helped me out a ton with this blog site (I’ll be sure to clear the cache before I call you up). Well…I already took a week to figure out more git commands (I will never fully master all these confusing instructions), reading both hugo docs and Razon’s docs and trying to get github pages to update my changes. However, I do plan on improving the website some more.
Now, it may be surprising to some of you guys out there but I actually have friends. Some of these friends might even go on this website and read these words as we speak. These friends, I message them quite often and frankly, I think they are getting annoyed of me sending them messages so damn often. With the creation of this blog site, I plan on bothering them even more. Send them more memes from Lemmy and posts about how Big Tech is coming after you and everybody is stealing your information, how you should convert to FOSS and use Linux instead.
Future plans
Even though I’ve slaved many days on just getting this site up and running, I still plan on adding more features to it.

Giscuss widget for comments
Uploading my documentations for projects I’ve done
Make some categories for personal life, gaming, school, rant, and whatnot
Whatever the heck pops into my mind

Stay tuned for more updates…

https://userliluzibert.github.io/news/2024/01/first-post/

**Comment by liluzibird** [comment]
Comment down here by signing in with your GitHub account.
